src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/DefaultTranscriptsList.tsx

```diff
@@ -25,7 +25,7 @@ import TranscriptsFilter from 'src/content/app/entity-viewer/gene-view/component
 import { TicksAndScale } from 'src/content/app/entity-viewer/gene-view/components/base-pairs-ruler/BasePairsRuler';
 import { Gene } from 'src/content/app/entity-viewer/types/gene';
 
-import { ReactComponent as Chevron } from 'static/img/shared/chevron-down.svg';
+import { ReactComponent as ChevronDown } from 'static/img/shared/chevron-down.svg';
```

Comment:
I am a little confused about how we handle the chevrons. We can have a single svg that we then rotate 360 degrees using CSS. We might have 2 svgs — one for up/down chevron and one for left/right chevron that we rotate 180 degrees using CSS, or we might have 4 different svgs (seems a bit wasteful, but whatever). At the moment, I don't think we even have a clear strategy. What you guys think? @imransl @msmanoj ?

Reply:
Just to clarify this file wasn't changed by jyo and is not related to that PR, i added this change for the Filter and sort PR and it is coming up in this PR because of the git commit prehook to fix the styling. I think having one chevron and rotating is a good idea, its just one file and i know the images/icons will increase and become a maze. But i suggest creating another ticket for this.

Reply:
It is better create a new ticket for the chevron icon functionality